cves/kernel/CVE-2022-36879.yml

```diff
@@ -268,8 +268,8 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: this vulnerability does not allow for the attacker to have access to other parts of the system or sensitive information. A possible exploit would only have the oppertuninity to cause a denial of service.
```

Comment:
How would they cause a denial of service? Good start!

Reply:
Thanks! Improved the note.